e2e/worker/.bazelrc

```diff
@@ -0,0 +1 @@
+build:verbose --define=VERBOSE_LOGS=1
```

Comment:
VERBOSE_LOGS?

Reply:
https://github.com/aspect-build/rules_ts/blob/b55b8608bcd4255c6b13459dea557e63ef6bf605/ts/private/ts_project.bzl#L74 
 
 tsc will spit out some helpful information that we use for debugging purposes.